src/templates/blogTemplate.js

```diff
@@ -0,0 +1,34 @@
+import { graphql } from 'gatsby'
+import React from 'react'
+
+export default function Template ({
+  data // this prop will be injected by the GraphQL query below.
+}) {
+  const { markdownRemark } = data // data.markdownRemark holds our post data
+  const { frontmatter, html } = markdownRemark
+  return (
+    <div className='blog-post-container'>
+      <div className='blog-post'>
+        <h1>{frontmatter.title}</h1>
+        <h2>{frontmatter.date}</h2>
+        <div
+          className='blog-post-content'
+          dangerouslySetInnerHTML={{ __html: html }}
+        />
+      </div>
+    </div>
```

Comment:
@andriawan Could you please style this page so that it looks like the other pages by utilizing `Layout` component?
Thanks!

Reply:
As you can see in [this preview](https://deploy-preview-75--devpendent.netlify.com/blog/sample-markdown-post), currently, it is not styled yet.

Reply:
okey, will do it as soon as possible @zainfathoni